Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -375,21 +358,15 @@ public void Dispose()
             }
         }
 
-        public class DummyToken : IDisposable
-        {
-            public void Dispose()
-            {
-            }
-        }
-
         public static IDisposable Read(this ReaderWriterLockSlim obj)
         {
             //if (BaseSqliteRepository.ThreadSafeMode > 0)
             //{
             //    return new DummyToken();
             //}
-            return new WriteLockToken(obj);
+            return new WriteLockToken(obj); // TODO: fix segfault
```

Comment:
this message should be expanded, something like "fix segfault if using Read/Write lock"

Reply:
I took a closer look at the db code and figured out the reason, I'll try and restructure all databases when my current big PRs are in.

Reply:
Still this TODO should be either reformulated to be clear or removed altogether.